README.md

```diff
@@ -29,7 +29,14 @@ project! Yay humans!
 
 [![Onboarding screenshot](http://i.imgur.com/JmFiMue.png)][onboarding]
 
-Also, [sign up](https://archivers-slack.herokuapp.com/) and join us on Slack! Active discussion is happening in the #webmonitoring channel with various offshoot topics happening on channels prefixed with #webmonitoring.
+### Where we work
+
+* Say hi on our chat!
+   * [Create](https://archivers-slack.herokuapp.com/) an account on Slack team.
+   * Join us in the `#webmonitoring` channel.
+* Attend a software development call.
+   * [Create](https://zoom.us/signup) a Zoom account.
```

Comment:
You don’t actually need an account! Just need to download the app. https://zoom.us/download (In case accounts are off-putting to people.)

Reply:
Oh, good call!